queue/queue.py

```diff
@@ -3,13 +3,16 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    self.storage.append(item)
   
   def dequeue(self):
-    pass
+    try:
+      return self.storage.pop(0)
+    except IndexError:
+      return None
 
```

Comment:
great work, see if you can implement a queue using a singly linked list ds, that way runtime will be constant

Reply:
A singly linked list would make appends less efficient, no? I thought of using a doubly linked list after I implemented one, but I didn't bother since I still had more to do. 😄